Sofa/Component/Constraint/Projective/tests/FixedConstraint_test.cpp

```diff
@@ -213,7 +212,6 @@ struct FixedConstraint_test : public BaseTest
         typename MechanicalObject::ReadVecCoord readX = dofs->readPositions();
 
         /// check info before topological changes
-        EXPECT_EQ(tCon->getPoints().size(), nbrDofs);
```

Comment:
Why deleting this line?

Reply:
`getPoints()` for PointSetTopologyContainer has been deleted.
https://github.com/sofa-framework/sofa/blob/f0db46804031b9508590960efb2e51a9cff9fb49/Sofa/Component/Topology/Container/Dynamic/src/sofa/component/topology/container/dynamic/PointSetTopologyContainer.h#L140

But I should have used getNbPoints() instead of deleting it